OrbitService/main.cpp

```diff
@@ -40,7 +46,17 @@ int main(int argc, char** argv) {
   std::string grpc_server_address = absl::GetFlag(FLAGS_grpc_server_address);
   uint16_t asio_port = absl::GetFlag(FLAGS_asio_port);
 
+  LinuxTracing::TracingOptions tracing_options;
+  tracing_options.trace_context_switches = GParams.m_TrackContextSwitches;
```

Comment:
This would only work on the client side, you can finally get rid of this.

Reply:
Sure.